testsuite/objects/__init__.py

```diff
@@ -82,6 +82,10 @@ def asdict(self):
             return {self.in_location: {"prefix": self.keySelector}}
         return {self.in_location: {"name": self.keySelector}}
 
+    def __eq__(self, other_location):
```

Comment:
I think `__eq__` should check all the vars, I would much rather prefer changing it in the tests